app/functions/functions.php

```diff
@@ -127,6 +129,13 @@ function submit_lost(){
 
 /* ###### Helper Functions ###### */
 
+// Print out variable if it exists, replace with default if not
 function print_tag($var, $default = '&mdash;') {
 	echo isset($var) ? $var : $default;
 }
+
+// Pretty URLs
+function parseUrl() {
+	// Trim URL, Sanitize and explode to array at each /
+	return $url = explode('/pet-details/', filter_var(rtrim($_SERVER['REQUEST_URI'], '/'), FILTER_SANITIZE_URL));
```

Comment:
For posterity turns out its GitHubs way of handling tabs vs spaces. Looks like 7 on mobile and 3 on desktop.